### DISCUSSION TITLE
Option to disable containers in a stack

### DISCUSSION THREAD

**Discussion by tekofx** [louislam/dockge#394] [Feature Request]
### ⚠️ Please verify that this feature request has NOT been suggested before.

- [X] I checked and didn't find similar feature request

### 🏷️ Feature Request Type

API, UI Feature

### 🔖 Feature description

Add option to disable containers in the stack. This way all the actions of starting, restarting, etc. will only affect the containers that are enabled.

### ✔️ Solution

In the list of containers show a button next to each container to disable it

**Comment by codersaur** [comment]
Good idea. You can use the profiles option to achieve this:

`    profiles: ["disabled"]`
